packages/e2e-tests/test-applications/create-next-app/tests/behaviour-client.test.ts

```diff
@@ -7,6 +7,7 @@ const sentryTestProject = process.env.E2E_TEST_SENTRY_TEST_PROJECT;
 const EVENT_POLLING_TIMEOUT = 30_000;
 
 test('Sends a client-side exception to Sentry', async ({ page }) => {
+  page.on('console', msg => console.log(msg.text()));
```

Comment:
Just to double check: Is this a leftover or do we want to keep this for better logs?

Reply:
nope, def. a leftover, oops 😅 
```suggestion
```